includes/admin/class-wc-admin-reports.php

```diff
@@ -110,6 +110,17 @@ public static function get_reports() {
 					),
 				),
 			),
+			'products'     => array(
+				'title'  => __( 'Products', 'woocommerce' ),
+				'reports' => array(
+					"downloads" => array(
+						'title'       => __( 'Product Downloads', 'woocommerce' ),
```

Comment:
Ah I see; I'll switch that.  I was basing this on the "Most Stocked" one so I followed the same case.  I'll change that to "Most stocked" while I'm in there.